runner/run_test.go

```diff
@@ -675,7 +675,7 @@ func TestFailedTestsRun(t *testing.T) {
 	assert.Equal(t, 1, res.Stats.TotalFailed())
 }
 
-func TestApplyInputOverrideSetHostFromDestAddr(t *testing.T) {
+func TestApplyInputOverrideEmptyHostHeaderSetHostFromDestAddr(t *testing.T) {
```

Comment:
Add another test for the default case, where `OverrideEmptyHostHeader` has not been set.